provisioning/miniprov/hare_mp/validate.py

```diff
@@ -0,0 +1,43 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+# Setup utility for Hare to configure Hare related settings, e.g. logrotate,
+# report unsupported features, etc.
+
+from hare_mp.store import ConfStoreProvider
+from hare_mp.store import ValueProvider
+
+class Validate:
+    def __init__(self, provider: ValueProvider):
+        super().__init__()
+        self.provider = provider
+
+    def is_first_node_in_storage_set(self) -> bool:
+        machine_id = self.provider.get_machine_id()
+        cluster_id = self.provider.get_cluster_id()
+        storage_set_id = self.get_storage_set_id()
+        server_nodes_key: str = (f'cluster>{cluster-id}>'
```

Comment:
`s/cluster-id/cluster_id` please

Reply:
ah, I did change this after testing, some how it is not updated in the patch.

Reply:
Done.